include/jwt-cpp/jwt.h

```diff
@@ -2755,6 +2768,11 @@ namespace jwt {
 		 * \return *this to allow for method chaining
 		 */
 		builder& set_issued_at(const date& d) { return set_payload_claim("iat", basic_claim<json_traits>(d)); }
+		/**
+		 * Set issued at claim to current time, as determined by std::chrono::system_clock::now()
+		 * \return *this to allow for method chaining
+		 */
+		builder& set_issued_now() { return set_issued_at(std::chrono::system_clock::now()); }
```

Comment:
For consistency, this should match the `verifier` and have a templated clock 

https://github.com/Thalhammer/jwt-cpp/blob/953aab6fe47be9e92fcbf5fb8418dee0d682e7ca/include/jwt-cpp/jwt.h#L3249-L3250

Reply:
This - you mean method or the `builder` class? If the clock is templated, shouldn't both `set_issued_now` and `set_expire_in` has the same clock source?

Reply:
The class, this way it's constructed with the same clock :) the verifier does that already

Reply:
Added a type parameter Clock to the builder

Doesn't compile at the moment tho, have to fix json traits now

Reply:
Done